README.md

```diff
@@ -7,20 +7,25 @@
 
 ## Introduction
 
-Uhyve is small hypervisor to boot the [Hermit kernel](https://github.com/hermitcore/kernel), which  is a unikernel operating system targeting a scalable and predictable runtime behavior for HPC and cloud environments.
+Uhyve is a small hypervisor specialized for the [Hermit kernel](https://github.com/hermitcore/kernel).
 
-**Warning:** At the moment Uhyve grants full host file system access from within the unikernel with the permissions of the user running Uhyve.
-Thus, it should not be used for applications which require isolation from the host system.
+> [!WARNING]
+> For the time being, Uhyve provides the unikernel full host file system access with the permissions of the user running Uhyve.
+> Thus, it should not be used for applications which require isolation from the host system.
 
 ## Installation
 
-An installation of the Rust toolchain is required.
-Please visit the [Rust website](https://www.rust-lang.org/) and follow the installation instructions.
+1. Install the Rust toolchain.
 
-Install Uhyve with
+> [NOTE]
```

Comment:
```suggestion
> [!NOTE]
```

Reply:
Do we need to highlight this as *note* at all? I think it is just a minor thing

Reply:
gahhhhhh, sorry for the misses! i'll adjust this in a sec